k8s/rke2-mc/templates/rke2-calico-config.template.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: helm.cattle.io/v1
+kind: HelmChartConfig
+metadata:
+  name: rke2-calico
+  namespace: kube-system
+spec:
+  valuesContent: |-
+    installation:
+      calicoNetwork:
+        bgp: Enabled
+        ipPools:
+        - blockSize: 24
+          cidr: 10.42.0.0/16
+          encapsulation: IPIP
+          natOutgoing: Enabled
```

Comment:
should be dynamic

Reply:
This file is for example

Reply:
but there's no reference to it anywhere and it's in the readme anyway